app/org/sagebionetworks/bridge/models/schedules/Schedule.java

```diff
@@ -146,6 +147,28 @@ public String getEventId() {
     public void setEventId(String eventId) {
         this.eventId = eventId;
     }
+    /**
+     * A persistent schedule is one that keeps a task alive in the list of tasks, 
+     * recreating it every time it is completed. Persistent schedules are scheduled to 
+     * occur one time, but have an event ID that immediately triggers re-scheduling when 
+     * one of the activities assigned by the schedule is completed.
+     * @return
+     */
+    public boolean getPersistent() {
+        if (activities != null && schedulesImmediatelyAfterEvent()) {
+            for (Activity activity : activities) {
+                if (activity.isPersistentlyRescheduledBy(this)) {
+                    return true;
+                }
+            }
+        }
+        return false;
+    }
+    public boolean schedulesImmediatelyAfterEvent() {
```

Comment:
Yes, I did it at both places so that no matter whether you query through the schedule, or through the activities, you get the same answer...